specification/_types/query_dsl/TextExpansionQuery.ts

```diff
@@ -17,11 +17,17 @@
  * under the License.
  */
 
+import { TokenPruningConfig } from './TokenPruningConfig'
 import { QueryBase } from './abstractions'
 
 export class TextExpansionQuery extends QueryBase {
   /** The text expansion NLP model to use */
   model_id: string
   /** The query text */
   model_text: string
+  /** Token pruning configurations
+   * @availability stack since=8.13.0 stability=experimental
+   * @availability serverless stability=experimental
```

Comment:
**Non-blocking**
Is the stability set to experimental based on this [discussion](https://groups.google.com/a/elastic.co/g/es/c/hQLKXHKYwxg/m/NgQbS8UuAwAJ)? Is there Elastic specific guide for the definition of **experimental**?

Reply:
Hi @saikatsarkar056 - could you please avoid linking to private materials on public PRs? I'm happy to discuss this question further offline with you if you would like.

Reply:
Sorry about that. I removed the link.